.travis.yml

```diff
@@ -21,6 +21,8 @@ node_js:
   - 6
   - 7
   - 8
+  - 9
+  - 10
```

Comment:
The latest node version (currently 10) is already tested at the `matrix` section (`node_js: node`), so there's no need to add it here (at least until version 11 is released)

Reply:
Thanks for letting me know!